server/src/main/java/com/example/server/businessLayer/Market/Shop.java

```diff
@@ -383,7 +383,7 @@ public synchronized void addEmployee(Appointment newAppointment) throws MarketEx
             }
     }
 
-
+    //todo mix those two methods.
     public void newAddEmployee(Appointment newAppointment) throws MarketException {
```

Comment:
refator this method:

- [ ] seems like both cases it creates an owner
- [ ] move owner creation and manager creation to external methods